src/test/java/steps/CurriculumSteps.java

```diff
@@ -0,0 +1,147 @@
+/*
+ * @(#) CurriculumSteps.java Copyright (c) 2019 Jala Foundation.
+ * 2643 Av. Melchor Perez de Olguin, Colquiri Sud, Cochabamba, Bolivia.
+ * All rights reserved.
+ *
+ * This software is the confidential and proprietary information of
+ * Jala Foundation, ("Confidential Information").  You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jala Foundation.
+ */
+
+package steps;
+
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+import project.ui.pages.*;
```

Comment:
Please use simple imports and not with the use of an asterisk.

Reply:
done